roles/tas_single_node/tasks/podman.yml

```diff
@@ -12,10 +12,16 @@
     - "{{ tas_single_node_kube_manifest_dir }}"
     - "{{ tas_single_node_kube_configmap_dir }}"
 
+- name: Check if Podman is already logged in to registry.redhat.io
+  ansible.builtin.command: podman login --get-login registry.redhat.io
+  register: logincheck
+  failed_when: logincheck.stdout == 'Error":" not logged into registry.redhat.io'
+  changed_when: logincheck.stdout | string != tas_single_node_registry_username | string
+
 - name: Podman login to registry.redhat.io
```

Comment:
You probably need to use explicit `changed_when: true` on this task to make ansible-lint happy. In this case is ok, because this task does change host state when it runs (and when it is skipped it won't be changed :))

Reply:
Ansible lint has been appeased!

Reply:
awating the molecule run through, which btw is awesome that we have this now.